Add docs about "updating Tobira" and "resync" (#598)
elan-ev/tobira: docs/docs/setup/resync.md

---
sidebar_position: 12
---

# Resync Opencast data

> Sometimes during Tobira development, we require more information about an event or series. (For example, an event's subtitles.)

Reviewer: ```suggestion
Sometimes during Tobira development, we require more information about an event or series. (For example an event's subtitles.)
```

> Then, we will (a) update the Tobira Harvest API inside Opencast to provide that data, and (b) update Tobira to store that data.

Reviewer: ```suggestion
To get it, we will (a) update the Tobira Harvest API inside Opencast to provide that data, and (b) update Tobira to store that data.
```

> However, events & series that have already been synced are not updated automatically and thus lack that additional new information.

Reviewer: ```suggestion
However, events and series that have already been synced are not updated automatically and thus lack that additional new information.
```

To fix that, you have to perform a manual "resync".
This is triggered by running `tobira sync reset`.

> Afterwards, the next time Tobira synchronizes (which is likely happening soon as part of your `tobira worker` process), it will synchronize all data.

Reviewer: ```suggestion
Afterwards, the next time Tobira synchronizes (which is likely happening soon as part of your `tobira worker` process), it will re-synchronize all data.
```
